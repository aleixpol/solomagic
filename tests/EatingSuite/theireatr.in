\ref something0
\mt somethingsomething andria somethingsomething

\ref something1
\mt somethingsomething andrian somethingsomething

