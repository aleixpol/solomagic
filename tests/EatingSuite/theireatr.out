\ref something0
\mt somethingsomething andria somethingsomething
\ma somethingsomething angria somethingsomething

\ref something1
\mt somethingsomething andrian somethingsomething
\ma somethingsomething andrian somethingsomething
